ROADMAP.md

```diff
@@ -199,16 +199,38 @@ Any dates listed below and the specific issues that will ship in a given milesto
 * Stacks Manager support for private repos and robot account credentials
 * Release process and efficiency improvements
 
+## [v0.10.0 backup/restore compatibility, resource composition, Open Application Model](https://github.com/crossplane/crossplane/releases/tag/v0.10.0)
+* Backup/restore support - e.g. with Velero
+  - Allow a KubernetesApplication to be backed up and restored [crossplane#1382](https://github.com/crossplane/crossplane/issues/1382)
+  - Allow connection secrets to be backed up and restored [crossplane-runtime#140](https://github.com/crossplane/crossplane-runtime/issues/140)
+  - Support backup and restore of all GCP managed resources [provider-gcp#207](https://github.com/crossplane/provider-gcp/issues/207)
+  - Support backup and restore of all Azure managed resources [provider-azure#128](https://github.com/crossplane/provider-azure/issues/128)
+  - Support backup and restore of all AWS managed resources [provider-aws#181](https://github.com/crossplane/provider-aws/issues/181)
+  - Allow Stack, StackInstall, StackDefinition to be backed up and restored [crossplane#1389](https://github.com/crossplane/crossplane/issues/1389)
+
+* Resource composition - experimental MVP [#1343](https://github.com/crossplane/crossplane/issues/1343)
+
+* Experimental support for [OAM](https://oam.dev/) (Open Application Model) API types
+  * Revised [Kubernetes-friendly OAM spec](https://github.com/oam-dev/spec/pull/304/files)
+  * OAM App Config Controller support [#1268](https://github.com/crossplane/crossplane/issues/1268)
+  * Enhance Crossplane to support a choice of local and remote workload scheduling
+  * OAM sample app: [crossplane/app-service-tracker](https://github.com/crossplane/app-service-tracker)
+
+* Docs overhaul (part 2/3) - https://crossplane.io/docs
+  * Documentation (and diagrams) about data model in Crossplane (including both application and infrastructure)
+  * Updated docs sidebar
+
+## v0.11.0
+* Docs overhaul (part 3/3) - https://crossplane.io/docs
+  * Documentation (and diagrams) about data model in Crossplane (including both application and infrastructure)
+  * Updated docs sidebar
+
```

Comment:
Might be worth adding point about name refactoring / any deprecation that may be made

Reply:
added